### ISSUE TITLE
Span with surrounding quotation marks appears as if without ones

### ISSUE THREAD

**Issue by cjer** [webanno/webanno#1033]
Span annotation with beginning or closing quotation mark (and I believe other punctuation marks as well) appears as if the punctuation is not included in the mark. This made annotators miss a lot of these small border issues.

## Examples:

`"הראל"` will visually appear the same as `הראל`, but as you can see they are annotated differently, as recognized by the curation interface.
![image](https://user-images.githubusercontent.com/9987721/43865035-0122c93c-9b6a-11e8-99ee-ede446eb850f.png)
![image](https://user-images.githubusercontent.com/9987721/43865083-27ffe1f2-9b6a-11e8-8786-c5c8d477420e.png)
![image](https://user-images.githubusercontent.com/9987721/43865048-0e8ef564-9b6a-11e8-8774-99f67cbedeae.png)
![image](https://user-images.githubusercontent.com/9987721/43865104-3cf20cb6-9b6a-11e8-915f-8207e9270211.png)

Same goes for `חטיבת "הראל"` and `חטיבת "הראל`

![image](https://user-images.githubusercontent.com/9987721/43864231-d3f3e916-9b67-11e8-9be9-022a9a7ba010.png)

**Comment by reckart** [comment]
Sounds like an edge-case. I assume the quotes are not unicode RTL characters, right? Can you verify whether the quotes are detected as separate tokens? So `"הראל"` should consist of three tokens. You could check that e.g. by exporting the data as TSV and seeing whether `"`, `הראל` and `"` all appear on separate lines. If they are, then we have an unknown bug. If they are not and this is a mixed RTL-LTR token, then it sounds like a known bug (https://github.com/webanno/webanno/issues/283).

**Comment by reckart** [comment]
Actually, re-reading https://github.com/webanno/webanno/issues/283 it sounds like the same thing.

**Comment by cjer** [comment]
Yes these are regular ASCII quotation marks. And yes, they are recognized as separate tokens.
